seed.js

```diff
@@ -38,7 +38,7 @@ async function makeAdmin() {
   let adminGroup = new Group({
     name: 'Admin2',
     slug: 'Admin2',
-    token: 'myAccessToken',
+    token: 'f5df675da94c4853e8c9e2a8bad21ac4198462c8',
```

Comment:
I'm not super versed on SR Connect, and I'm not entirely certain what this function does. But is ok to store a token in a public repo?

Reply:
The token is randomly generated. In this case, it's just a fake token for a test group. The original `myAccessToken` failed validation causing the group creation to fail. However, @DylanOrsh could we just generate this value on the fly: `token: makeRandomString(40),`?